<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>429</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>587</width>
    <height>429</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Canada存档器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>备份存档</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(248, 248, 248);
    border-bottom: 6px solid;
	 border-bottom-color: rgb(196, 198, 198)
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
    border-bottom: 10px solid;
	 border-bottom-color: rgb(192, 42, 43)
}</string>
         </property>
         <property name="text">
          <string>存档</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/number-1.png</normaloff>:/number-1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="shortcut">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(248, 248, 248);
    border-bottom: 6px solid;
	 border-bottom-color: rgb(196, 198, 198)
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
    border-bottom: 10px solid;
	 border-bottom-color: rgb(192, 42, 43)
}</string>
         </property>
         <property name="text">
          <string>存档</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/number-two.png</normaloff>:/number-two.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonSave3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(248, 248, 248);
    border-bottom: 6px solid;
	 border-bottom-color: rgb(196, 198, 198)
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
    border-bottom: 10px solid;
	 border-bottom-color: rgb(192, 42, 43)
}</string>
         </property>
         <property name="text">
          <string>存档</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/number-3.png</normaloff>:/number-3.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>16</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>读取存档</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLoad1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(248, 248, 248);
    border-bottom: 6px solid;
	 border-bottom-color: rgb(196, 198, 198)
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
    border-bottom: 10px solid;
	 border-bottom-color: rgb(34, 191, 242)
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/number-one.png</normaloff>:/number-one.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLoad2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(248, 248, 248);
    border-bottom: 6px solid;
	 border-bottom-color: rgb(196, 198, 198)
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
    border-bottom: 10px solid;
	 border-bottom-color: rgb(34, 191, 242)
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/number-2.png</normaloff>:/number-2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLoad3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(248, 248, 248);
    border-bottom: 6px solid;
	 border-bottom-color: rgb(196, 198, 198)
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
    border-bottom: 10px solid;
	 border-bottom-color: rgb(34, 191, 242)
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/number-three.png</normaloff>:/number-three.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>存档路径:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditHomePath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOpenFile">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(228, 228, 228);
   
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
   border: 3px solid;
}</string>
       </property>
       <property name="text">
        <string>打开</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>僵点子点数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditZPoint">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonChangeZPoint">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>10</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
}

QPushButton:hover {
   background-color: rgb(228, 228, 228);
   
}

QPushButton:pressed {
   background-color: rgb(248, 248, 248);
   border: 3px solid;
}</string>
         </property>
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonAbout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
	border: 1px solid;
	border-color:rgb(200,200,200);
}

QToolButton:hover {
   background-color: rgb(228, 228, 228);
	border: 2px solid;
	border-color:rgb(200,200,200);
   
}

QToolButton:pressed {
   background-color: rgb(248, 248, 248);
   border: 2px solid;
}</string>
       </property>
       <property name="text">
        <string>关于</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/info.png</normaloff>:/info.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonGitHub">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
    border-radius: 10px;
    background-color: rgb(248, 248, 248);
	border: 1px solid;
	border-color:rgb(200,200,200);
}

QToolButton:hover {
   background-color: rgb(228, 228, 228);
	border: 2px solid;
	border-color:rgb(200,200,200);
   
}

QToolButton:pressed {
   background-color: rgb(248, 248, 248);
   border: 2px solid;
}</string>
       </property>
       <property name="text">
        <string>GitHub</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/github.png</normaloff>:/github.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
